cmake/templates/HPXConfig.cmake.in

```diff
@@ -42,8 +42,8 @@ set(HPX_Fortran_COMPILER "@CMAKE_Fortran_COMPILER@" CACHE STRING "Fortran compil
 set(HPX_Fortran_COMPILER_ID "@CMAKE_Fortran_COMPILER_ID@" CACHE STRING "Fortran compiler id for HPX" FORCE)
 set(HPX_Fortran_COMPILER_VERSION "@CMAKE_Fortran_COMPILER_VERSION@" CACHE STRING "Fortran compiler version for HPX" FORCE)
 
-set(HPX_BOOST_ROOT "@BOOST_ROOT@")
-set(HPX_BOOST_LIBRARIES "@Boost_LIBRARIES@")
+set(BOOST_ROOT "@BOOST_ROOT@")
+include(HPX_SetupBoost)
```

Comment:
Wouldn't this re-detect Boost instead of forcing dependent libraries into using the very same version that was used for compiling HPX itself?

Reply:
@aurianer what should we do about this one? Does it actually require supplying `BOOST_ROOT` again when configuring projects using HPX?

Reply:
@hkaiser I think the answer is that yes, it will re-detect Boost and that seems to be the way to do it in modern CMake.